service/build.gradle

```diff
@@ -84,6 +84,9 @@ dependencies {
     // sentry
     implementation 'io.sentry:sentry:7.3.0'
 
+    // jersey hk2 for dependency injection
+    implementation 'org.glassfish.jersey.inject:jersey-hk2'
```

Comment:
was seeing a lot of 
```
Jersey-HK2 module is missing. Falling back to injection-less client. Injection may not be supported on the client.
```
warn logs in our logs that this should get rid of for seemingly no downside?

Reply:
thank you for addressing this!